kotest-property/src/commonMain/kotlin/io/kotest/property/arbitraries/arbitraries.kt

```diff
@@ -0,0 +1,25 @@
+package io.kotest.property.arbitraries
```

Comment:
`property` is singular, perhaps `arbitrary` should be too.

Reply:
Done.